randovania/games/fusion/logic_database/Sector 1 SRX.txt

```diff
@@ -293,12 +293,26 @@ Extra - room_id: [9]
   > Door to Checkpoint
       After Stabilizer 5 Online
   > Event - Stabilizer 5 Online
-      All of the following:
-          # TODO: redo this. If using charge, you dont need to damage boost. You do need it for Missiles tho. But you dont for screw/PBs. Also add way to only kill with 1 missile.
-          Charge Beam or Can Kill Missile Geron
-          Any of the following:
+      Any of the following:
+          # Charge Beam can kill the Geron because the beam cannon can be stuck inside the shell and fire past it without touching the shell
+          Charge Beam and Knowledge (Intermediate)
+          All of the following:
+              # Vanilla - Morph Ball to cross to the other side of the tunnel and use Missiles or Charge Beam
               Morph Ball
-              Damage Boosts (Beginner) and Normal Damage ≥ 20
+              Charge Beam or Missiles
+          All of the following:
+              Any of the following:
+                  # Missiles can be used to damage the Geron from the wrong side, but only by getting too close
+                  All of the following:
+                      Knowledge (Intermediate)
+                      Any of the following:
+                          Damage Boosts (Intermediate) and Normal Damage ≥ 90 and 10 DMG Missiles
+                          Damage Boosts (Intermediate) and Normal Damage ≥ 60 and 20 DMG Missiles
+                          Damage Boosts (Beginner) and Normal Damage ≥ 30 and 30 DMG Missiles
```

Comment:
Is it worthwhile to have all the requirements if you can also farm health off of the Geron?

Reply:
Probably not 😄, can trim this down a bit